examples/lines_bars_and_markers/stackplot_demo.py

```diff
@@ -12,29 +12,26 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-# Fixing random state for reproducibility
-np.random.seed(19680801)
+x = [1, 2, 3, 4, 5]
+y1 = [1, 1, 2, 3, 5]
+y2 = [0, 4, 2, 6, 8]
+y3 = [1, 3, 5, 7, 9]
 
+y = np.vstack([y1, y2, y3])
 
-def fnx():
-    return np.random.randint(5, 50, 10)
-
-
-y = np.row_stack((fnx(), fnx(), fnx()))
-x = np.arange(10)
-
-y1, y2, y3 = fnx(), fnx(), fnx()
+labels = ["fib", "evens", "odds"]
```

Comment:
Might as well spell the first one out; it's only clearly Fibonacci to someone who knows what the Fibonacci sequence is.